### DISCUSSION TITLE
mater 的raft任期不一致

### DISCUSSION THREAD

**Discussion by TheTian-Sun** [cubefs/cubefs#3398] [Q&A]
使用helm在k8s上部署了cubefs, 由于机房断电导致了服务重启，重启后master 0和1在任期12,master0 为leader，master 2处于任期11，导致master2不可用，怎么使master2恢复到可用状态

**Comment by leonrayang** [comment]
@TheTian-Sun 正常情况下，主从主从raft通讯恢复，master 2的term会根据leader的升高，日志会同步，不需要特殊处理。需要确认网络是否恢复

**Comment by TheTian-Sun** [comment]
新任期已开始，数据已同步，感谢回复



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年6月14日(周五) 下午4:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [cubefs/cubefs] mater 的raft任期不一致 (Discussion #3398)




 
@TheTian-Sun 正常情况下，主从主从raft通讯恢复，master 2的term会根据leader的升高，日志会同步，不需要特殊处理。需要确认网络是否恢复
